### ISSUE TITLE
PX4FLOW and PixHawk 4 mini issue

### ISSUE THREAD

**Issue by acpl00** [PX4/Flow#112]
Dear people.

I have set up:

-PixHawk 4 mini
-PX4FLOW V2.3
-GPS

The communication to the groundcontrol station is via wireless.

So far I can see in the mavlink inspector the:

  -optical_flow_rad 
  -local_position_ned

I use EKF2 and I have Set

 SENS_FLOW_ROT to 270 degrees, default 
 EKF2_AID_MASK to use optical flow 
 EKF2_HGT_MODE to use the range finder

The question is:

How can I check the the PX4FLOW sensor is working correctly together with the GPS?

In other words, I guess that I shall get a position [xyz] that comes from the combination of both the GPS and the PX4FLOW, right?

Any advise or suggestion is appreciated

**Comment by mike239x** [comment]
Hm, afaik PX4FLOW itself does not do anything with GPS, so any kind of combination with GPS happens on the PixHawk side, so you better ask there.
